<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>702</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>473</width>
    <height>702</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>473</width>
    <height>702</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:25px;
background-color: rgb(114, 159, 207);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="close_button">
     <property name="geometry">
      <rect>
       <x>416</x>
       <y>16</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Çıkış</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius:5px;
background-color: rgb(164, 0, 0);
}
QPushButton:hover{
background-color: rgb(164, 50, 150);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="secret_button">
     <property name="geometry">
      <rect>
       <x>394</x>
       <y>16</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Ekranı Gizle</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius:5px;
background-color: rgb(245, 121, 0);
}
QPushButton:hover{
background-color: rgb(245, 121, 160);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>172</x>
       <y>20</y>
       <width>121</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chakra Petch</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Not Defterim</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>42</y>
       <width>451</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>*****************************************************************************************************</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>640</y>
       <width>421</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(211, 215, 207);
</string>
     </property>
     <property name="text">
      <string>CREATED: By Selim Talha Çağlar - @Copyright 2021</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>411</width>
       <height>571</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chakra Petch</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 101, 164);
border-radius:5px;
color: rgb(211, 215, 207);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="not_ekle">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>16</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chakra Petch</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Not Ekle</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius:5px;
background-color: rgb(186, 189, 182);
}
QPushButton:hover{
background-color: rgb(186, 189, 232);
}</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="not_kaldir_button">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>16</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chakra Petch</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Not Sil</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius:5px;
background-color: rgb(186, 189, 182);
}
QPushButton:hover{
background-color: rgb(186, 189, 232);
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="secret_not">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>16</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chakra Petch</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Gizli Notları Göster</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: None;

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="public_not">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>16</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chakra Petch</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Normal Notları Göster</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius:5px;
background-color: None;
}
QPushButton:hover{
background-color: rgb(186, 189, 232);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>473</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
